src/lib/api.ts

```diff
@@ -41,19 +54,22 @@ export const makeFilterUrl = (path: string, query: URLSearchParams) => {
 	};
 };
 
-export const loadDataset: Load = async ({ params }) => {
+export const loadDataset: Load = async ({ params, fetch }) => {
 	const { datasetId } = params;
 	const dataset = await get(`package_show?id=${datasetId}`);
 	const showcases = await get(`ckanext_package_showcase_list?package_id=${datasetId}`);
+	const res = await fetch(schemaOrgRoute(datasetId));
```

Comment:
sveltekit will treat a local url so that it does not go to the remote side: therefore it will use the endpoint that we added and that endpoint will then go and fetch the data at the remote ckan backend.